test/tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
 	"compilerOptions": {
-		"baseUrl": "test",
+		"baseUrl": "",
```

Comment:
What causes this change? Isn't `baseUrl` relevant for how `paths` are looked up? Why does it not need to be "test" anymore?

Reply:
I was afraid you'd ask that 😄 I'm not 100% sure why this change is needed. But it seems to be related to the way the compiler options are parsed:

```
	const { options } = parseJsonConfigFileContent(
		configFile.config,
		sys,
		dirname(configPath)
	);
```

I have to pass a `basePath` to `parseJsonConfigFileContent` as the third argument. I didn't want to introduce a new option to pass the project root, so I decided to use the location of the `tsconfig.json` as the base path which is supported by [this comment](https://github.com/microsoft/TypeScript/issues/5276#issuecomment-148786708).

As the parsed compiler options are passed to `nodeModuleNameResolver` it already resolves the correct path without having to define `test` as baseUrl.

I guess before the PR the project root was `/`, but by passing the location of the `tsconfig.json` to `parseJsonConfigFileContent`, the root is now `/test` - at least from TypeScript's point of view.

Reply:
Ok I need to look into this again (tomorrow) because I just want to be aware whether this is a breaking change or not. Thanks for the explanation!

Reply:
No hurry. After implementing it, I realized that it doesn't help with the issue I'm having 😄